app/controllers/application_controller.rb

```diff
@@ -10,11 +10,7 @@ class ApplicationController < ActionController::API
   def with_current_request
     ManageIQ::API::Common::Request.with_request(request) do |current|
       begin
-        if Tenant.tenancy_enabled?
-          ActsAsTenant.with_tenant(current_tenant(current.user)) { yield }
-        else
-          ActsAsTenant.without_tenant { yield }
-        end
+        ActsAsTenant.with_tenant(current_tenant(current.user)) { yield }
```

Comment:
This looks like it makes us completely unable to bypass tenancy with `BYPASS_TENANCY=true` when running rails - is this the desired behavior?

Reply:
@lindgrenj6 We should never bypass tenancy, it's baked into the product, when we create records we can specify the tenant_id or when we run with the context request, it sets the current tenant in the above code.

Reply:
Another point to make here is that its dead code even in our specs, here is a snapshot of the function, the red line indicating the code never got even a single hit

<img width="612" alt="Screen Shot 2019-04-13 at 12 10 32 AM" src="https://user-images.githubusercontent.com/6452699/56059057-cf804d80-5d80-11e9-9aac-395a45f97c6a.png">